Dockerfile

```diff
@@ -24,6 +24,8 @@ WORKDIR /usr/src/app
 # Copying this separately prevents re-running npm install on every code change.
 COPY package.json package*.json ./
```

Comment:
Could be moved before line 42, which is the first time it is used.

Reply:
Also, 44 could be moved before both of these.

(just nitpicks for docker performance, not crucial)